app/views/devise/shared/_password_strength.html.slim

```diff
@@ -10,3 +10,4 @@
       span#pw-strength-txt.bold = '...'
     .h6
       #pw-strength-feedback.italic
+#forbidden-passwords*{'data-forbidden-passwords'=>@forbidden_passwords.join(',')}
```

Comment:
I'd put the data attribute on some existing element somewhere rather than creating an empty div